trl/trainer/ppo_trainer.py

```diff
@@ -644,6 +644,7 @@ def batched_forward_pass(
         queries: torch.Tensor,
         responses: torch.Tensor,
         model_inputs: dict,
+        return_logits: bool = False,
```

Comment:
Could you add a sentence stating what this arg does below on the docstring

Reply:
Sure!
How about this sentence as the docstring text?

```
return_logits (`bool`, *optional*, defaults to `False`):
    Whether to return all_logits. Set to `False` if logits are not needed to reduce memory consumption.
```

Reply:
Awesome, this looks great!

Reply:
Thanks, I added the docstring!

Reply:
Great thank you!